umh.docs.umh.app/content/en/docs/production-guide/administration/access-database.md

```diff
@@ -30,15 +30,16 @@ you just want to see the data in a dashboard and don't need to manupulate it.
 ### Get the database credentials
 
 If you are not using the CLI, you need to know the database credentials. You can
-find them in the **{{< resource type="secret" name="db-psw">}}** Secret. By
-default, the username is factoryinsight and the password is changeme.
+find them in the **{{< resource type="secret" name="db-psw">}}** Secret. Run the
+following command to get the credentials:
 
 ```bash
```

Comment:
This is used I think in different places now. Previously I added an test that checks if this works. Can you modify the test as well that checks if this works? Additionally, can you ensure through comments etc. that any changes here must pass the automated test?

Reply:
`<!-- This command is tested within #1153 -->`